code/modules/mob/dead/observer/observer.dm

```diff
@@ -758,6 +766,37 @@ This is the proc mobs get to turn into a ghost. Forked from ghostize due to comp
 		to_chat(src, span_notice("Data HUDs enabled."))
 		data_huds_on = 1
 
+/mob/dead/observer/verb/unlock_antag_sight()
+	set category = "Ghost"
+	set name = "Unlock Antag HUD"
+	if(!client)
+		return
+	if(CONFIG_GET(flag/disable_ghost_unlock_antag_hud))
+		to_chat(src, span_warning("Unlocking the Antag HUD is currently disabled."))
+		return
```

Comment:
Can we make the verb itself conditional on the config being enabled? Is that a power we have with Byond?

Reply:
> Is that a power we have with Byond?

yes, you make it a proc instead of a verb and then add the proc to the mob's verb list during initialize.

to complete it, you also have to add code to the config item so if admins reload config mid round or vv the config, the verb is added to all existing ghosts.

Reply:
I could if it was a proc with `verbs +=` on init I suppose.